src/ImageSharp/Formats/Tiff/PhotometricInterpretation/CieLabTiffColor{TPixel}.cs

```diff
@@ -31,8 +31,8 @@ public override void Decode(ReadOnlySpan<byte> data, Buffer2D<TPixel> pixels, in
             for (int x = 0; x < pixelRow.Length; x++)
             {
                 float l = (data[offset] & 0xFF) * 100f * Inv255;
-                var lab = new CieLab(l, (sbyte)data[offset + 1], (sbyte)data[offset + 2]);
-                var rgb = ColorSpaceConverter.ToRgb(lab);
+                CieLab lab = new(l, (sbyte)data[offset + 1], (sbyte)data[offset + 2]);
```

Comment:
I'm curious. Why are we casting to `sbyte` here? CieLab takes `float` for the constructor parameters. 

The `L` component ranges from `0-100`
Generally, the `A` and `B` components are unbounded though it's common to clamp them to `−128` to `127`

_Note. I just realized the property notes on the type are incorrect, saying `-100` to `100` for the `A` and `B` components._

https://en.wikipedia.org/wiki/CIELAB_color_space